sapi_manifests/mako/template

```diff
@@ -5,14 +5,6 @@ worker_processes {{MAKO_WORKER_PROCESSES}};
 worker_processes  8;
 {{/MAKO_WORKER_PROCESSES}}
 
-{{#MAKO_THREAD_POOL_SIZE}}
-thread_pool mpu threads={{MAKO_THREAD_POOL_SIZE}};
-{{/MAKO_THREAD_POOL_SIZE}}
-{{^MAKO_THREAD_POOL_SIZE}}
-thread_pool mpu threads=8;
-{{/MAKO_THREAD_POOL_SIZE}}
-
-
 error_log /var/log/mako-error.log info;
 
 events {
```

Comment:
So... the MPU addition to this repo (the change was for 5 tickets, so it doesn't help tracking down the reference), added the:

```
use eventport;
```

line below.   MANTA-3114's description includes 

{quote}
Run with configurations that leverage event ports so we can use the thread pool API.
{quote}

I'm guessing that our usage of nginx after this may not need the thread pool API, so potentially that "use eventport;" could be dropped? Dunno.

https://trac.nginx.org/nginx/ticket/1226 isn't comforting. Nor https://www.ruby-forum.com/t/nginx-solaris-eventport/235625/2

Anyway, you aren't changing it here.

Reply:
Yeah, it *probably* could go away? I don't know. It works as is though, so I'll leave that for someone to investigate at a later date.

Thanks for the review!